code/modules/modular_computers/file_system/programs/engineering/engineering_camera.dm

```diff
@@ -0,0 +1,25 @@
+/datum/computer_file/program/camera_monitor/engineering
```

Comment:
sub-programs like this seem like a really messy way of handling it. Especially if you're not making specific ones for every department. 

There's supply and research camera monitors at least, which i don't see here. 

Why not only make one single subprogram - security, which has all cameras (for the telescience scenario i mentioned earlier). And for every other application just use the default one which comes with cameras based on the user's ID?